agent/agent-bootstrap/src/test/resources/applicationinsights.json

```diff
@@ -0,0 +1,60 @@
+{
+  "connectionString": "InstrumentationKey=00000000-0000-0000-0000-000000000000",
+  "role": {
+    "name": "Something Good",
+    "instance": "xyz123"
+  },
+  "customDimensions": {
+    "some key": "abc",
+    "another key": "def"
+  },
+  "sampling": {
+    "traceIdBased": {
```

Comment:
what is that? no longer fixedRate?

Reply:
it's the same underlying sampler, just new name to align with OpenTelemetry

i'm not 100% sure on this name, still have it on list to review shortly: https://msazure.visualstudio.com/One/_boards/board/t/Azure%20Attach%20(Java%20NET)/Features/?workitem=8657761

Reply:
can we leave it out to simplify:

`"sampling": {
    "probability": 10
 }`

Reply:
traceIdBased is more like implementation detail.. it doesn't bring clarity to the users...

Reply:
simplified based on your suggestion 👍